apps/nextjs-website/src/lib/strapi/homepage.ts

```diff
@@ -102,6 +112,25 @@ const makeStrapiHomepagePopulate = () =>
           'webinarSpeakers.avatar',
         ],
       },
+      ecosystem: {
+        populate: [
+          'products.logo',
+          'solutions.icon',
+          'solutions.steps',
+          'solutions.steps.products',
+          'solutions.steps.products.logo',
+          'solutions.stats',
+          'solutions.webinars',
+          'solutions.webinars.coverImage.image',
+          'solutions.webinars.relatedLinks.link',
+          'solutions.webinars.webinarSpeakers.avatar',
+          'solutions.bannerLinks',
+          'solutions.bannerLinks.icon',
+          'solutions.products',
+          'solutions.products.logo',
+          'solutionsCta.link',
+        ],
```

Comment:
I got an error when fetching data due to this line:
![Screenshot from 2024-06-25 16-31-40](https://github.com/pagopa/developer-portal/assets/24475319/17f5a4cc-fba3-4824-8a83-6704e287ad1a)
```suggestion
        populate: [
          'products.logo',
          'solutions.icon',
          'solutions.steps',
          'solutions.steps.products',
          'solutions.steps.products.logo',
          'solutions.stats',
          'solutions.webinars',
          'solutions.webinars.coverImage.image',
          'solutions.webinars.relatedLinks.links',
          'solutions.webinars.webinarSpeakers.avatar',
          'solutions.bannerLinks',
          'solutions.bannerLinks.icon',
          'solutions.products',
          'solutions.products.logo',
          'solutionsCta.link',
        ],
```

Reply:
The suggestion above should handle the problem, but i think that the issue is more complex is due to use a complete codec of solutions event when we need to fetch few data.  

NB: I suggest to resolve [this issue before](https://github.com/pagopa/developer-portal/pull/910/files#r1653011953)

I think is better only to fetch de need data and remove all unecessary includes:
```suggestion
        populate: ['products.logo', 'solutions.icon', 'solutionsCta.link'],
```